sig-docs/charter.md

```diff
@@ -0,0 +1,54 @@
+# SIG Docs Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and adheres to the Roles and Organization Management specified in [sig-governance].
+
+## Scope
+
+SIG Docs is responsible for Kubernetes documentation.
+
+The responsibility for feature documentation lies with feature developers. SIG Docs sets standards for feature documentation, provides clear paths for docs contribution, and coordinates documentation updates during quarterly releases.
+
+We maintain the Kubernetes website's infrastructure and tooling.
+
+We create subprojects as needed to handle specific aspects of Kubernetes documentation.
+
+### In scope
+
+SIG [readme]
+
+#### Code, Binaries and Services
+
+- The kubernetes.io website, infrastructure and tooling
+
+#### Cross-cutting and Externally Facing Processes
+
+- Standards for content: contribution and reviews
+- Coordinating docs contributions for quarterly releases
+
+### Out of scope
+
+- Primary documentation of new features
```

Comment:
What does this mean?  Can we update this with an example and who is responsible for this?

Reply:
I'm assuming this is referring to Alpha features. If so, then we should use that terminology so that it's clear.

Reply:
I read this as "we're not going to write the content for you, we are here to make sure it is legible and meets our standards"

Reply:
I think this still needs to be more clearly defined.

Reply:
I clarified this to read:

> SIG Docs is not responsible for documenting new technical content.